string object_class
---
bool detected
